src/plugins/intel_cpu/src/infer_request.cpp

```diff
@@ -153,6 +153,7 @@ void ov::intel_cpu::MKLDNNInferRequestBase::InferImpl() {
     graph = &(graphLock._graph);
 
     ThrowIfCanceled();
+    convertBatchedInputBlobs();
```

Comment:
can we write our own `convertBatchedInputBlob` where we will be perform copying in parallel and will not perform checks on remote context?
we can extract part with desc creation https://github.com/openvinotoolkit/openvino/blob/master/src/inference/src/cpp_interfaces/interface/ie_iinfer_request_internal.cpp#L302-L307 and copying https://github.com/openvinotoolkit/openvino/blob/master/src/inference/src/cpp_interfaces/interface/ie_iinfer_request_internal.cpp#L328-L348 in separate funcitons

Reply:
Fixed in 386174a4dcb05d0702858df276af4689cc0e3541. Thank you!